--- a/dsqss/dsqss-1.1.17/src/io.h
+++ b/dsqss/dsqss-1.1.17/src/io.h
@@ -143,7 +143,7 @@
   };
 
   bool read() {
-    bool b = INS.getline( LINE, BLEN);
+    bool b = static_cast<bool>(INS.getline( LINE, BLEN));
     if ( b ) { IL++; }
     return b;
   };
--- a/pmwa/pmwa-1.1.3/src/inc/xml.hpp
+++ b/pmwa/pmwa-1.1.3/src/inc/xml.hpp
@@ -62,7 +62,7 @@
   };
 
   bool read() {
-    bool b = INS.getline( LINE, BLEN);
+    bool b = static_cast<bool>(INS.getline( LINE, BLEN));
     if ( b ) { IL++; }
     return b;
   };
